src/fu-engine.c

```diff
@@ -7212,22 +7210,20 @@ fu_engine_load_plugins(FuEngine *self,
 
 	/* search */
 	plugin_path = fu_path_from_kind(FU_PATH_KIND_LIBDIR_PKG);
-	filenames = fu_path_get_files(plugin_path, error);
+	filenames = fu_path_get_files(plugin_path, &error_local);
 	if (filenames == NULL)
-		return FALSE;
+		g_debug("no external plugins found: %s", error_local->message);
 	fu_progress_step_done(progress);
 
 	/* load */
-	fu_engine_load_plugins_filenames(self, filenames, fu_progress_get_child(progress));
+	if (filenames != NULL)
+		fu_engine_load_plugins_filenames(self, filenames, fu_progress_get_child(progress));
 	fu_progress_step_done(progress);
 
 	/* load builtins */
 	if (flags & FU_ENGINE_LOAD_FLAG_BUILTIN_PLUGINS)
 		fu_engine_load_plugins_builtins(self, fu_progress_get_child(progress));
 	fu_progress_step_done(progress);
-
-	/* success */
-	return TRUE;
```

Comment:
personally I think keep the GError+gboolean -- we might need it later. It's okay to ignore it.